cromwellApiClient/src/test/scala/cromwell/api/CromwellResponseFailedSpec.scala

```diff
@@ -15,7 +15,7 @@ import scala.language.postfixOps
 
 class CromwellResponseFailedSpec extends TestKit(ActorSystem()) with AsyncFlatSpecLike with Matchers with BeforeAndAfterAll {
   override def afterAll(): Unit = {
-    Await.ready(system.terminate(), 1 second)
+    Await.ready(system.terminate(), 10 second)
```

Comment:
Could time dilation be useful here?

Reply:
Perhaps, what would that look like/mean?